wordpress-mobile/AztecEditor-iOS
# Make pre mergeable

Fixes #575 

This PR make sure that a `pre` that spreads multiple lines is processed correctly and kept inside a single element. It also make sures that when editing an existing pre that it's not separated in multiple elements.

To test:
 - Start the demo app
 - Open the demo with content for standard HTML
 - Check that the PRE element is processed and displayed correctly
 - Switch to HTML and check there is still a single PRE
 - Switch to visual mode and write inside the PRE and add a new line
 - Switch to HTML mode and see if it's still a single PRE.
